contracts/payment/PullPayment.sol

```diff
@@ -31,11 +31,31 @@ contract PullPayment {
      * This means that contracts unaware of the `PullPayment` protocol can still
      * receive funds this way, by having a separate account call
      * {withdrawPayments}.
+     *
+     * NOTE: This function has been deprecated, use {withdrawPaymentsWithGas}
```

Comment:
```suggestion
     * CAUTION: This function has been deprecated. Use {withdrawPaymentsWithGas}
```

Reply:
Is `CAUTION` the same as `WARNING`?

Reply:
Yes. In fact, it will say "Warning" too. But we can change this in the CSS later.

Reply:
I don't want both the deprecated and non-deprecated versions to have the same style of warning. Would it be possible to create a custom one for deprecations?